### ISSUE TITLE
burnin path change

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#482]
Originally reported of Google Code with ID 426.

```
Burnin was moved from /usr/lib/ganeti/tools/burnin to /usr/lib/ganeti/burnin in 2.7. This should be fixed, if possible.
```
Originally added on 2013-04-11 20:00:25 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit 7ee2098e8b326d75f0979a0e35f4b63ad7d1799a
Author: Guido Trotter <ultrotter@google.com>
Date:   Thu Apr 11 16:23:36 2013 -0400

    Fix burnin install path
    
    Due to a bug in 8e55e20f burnin's installation path got changed.
    Moving tools/burnin to the right makefile target reverts this (while
    preserving the value of the original commit).
    
    Signed-off-by: Guido Trotter <ultrotter@google.com>
    Reviewed-by: Helga Velroyen <helgav@google.com>

```
Originally added on 2013-04-11 20:43:15 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:49 +0000 UTC.

Changed State: Released
